### DISCUSSION TITLE
Please how do i use clarinet lsp in an editor

### DISCUSSION THREAD

**Discussion by Muizzyranking** [stx-labs/clarinet#1610] [Q&A] [answered]
I am using neovim and I am about to set up clarity lsp and then I saw that it has been archived since 2023 and the readme said there is lsp implementation on here. I do not see any way to use this.

**Comment by hugoclrd** [comment, marked as answer]
Hello @Muizzyranking 
There's an attempt here: https://github.com/aldur/clarity.nvim/

Although there are still some known issues when working with the LSP in neovim or some others editors. Let me know how it goes

**Comment by Muizzyranking** [comment]
I found out the problem was setting clarity as a supported server. Seems buggy but it works now. I hope for future improvements.
Thank you for your response.
